classes/Visualizer/Module/Chart.php

```diff
@@ -59,6 +59,7 @@ public function __construct( Visualizer_Plugin $plugin ) {
 		$this->_addAjaxAction( Visualizer_Plugin::ACTION_CLONE_CHART, 'cloneChart' );
 		$this->_addAjaxAction( Visualizer_Plugin::ACTION_EXPORT_DATA, 'exportData' );
 
+		error_log( var_export( 'REGISTER ACTION', true ) );
```

Comment:
NIT. You can remove this if it is just for debugging.

Reply:
Done, thank you!